src/Features/LanguageServer/Protocol/Handler/CodeActions/CodeActionResolveData.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.Handler.CodeActions
+{
+    /// <summary>
+    /// Provides data needed to resolve code actions.
+    /// </summary>
+    internal class CodeActionResolveData
+    {
+        /// <summary>
+        /// The unique title of a code action. No two code actions should
```

Comment:
i wouldn't call this a 'title' then.  'title' made me think this is just the "Generate Method" portion of the code action.  This really is a UniqueIdentifier sort of thing.